src/main/java/mate/academy/spring/controller/ShoppingCartController.java

```diff
@@ -31,14 +32,19 @@ public ShoppingCartController(ShoppingCartService shoppingCartService,
     }
 
     @PutMapping("/movie-sessions")
-    public void addToCart(@RequestParam Long userId, @RequestParam Long movieSessionId) {
+    public void addToCart(Authentication authentication, @RequestParam Long movieSessionId) {
```

Comment:
does it work as it is? or should we put `@RequestParam Long movieSessionId` first?